src/wp-includes/compat.php

```diff
@@ -374,7 +374,44 @@ function is_iterable( $var ) {
 		return ( is_array( $var ) || $var instanceof Traversable );
 	}
 }
+if ( ! function_exists( 'array_key_first' ) ) {
+	/**
+	 * Polyfill for array_key_first() function added in PHP 7.3.
+	 *
+	 * Get the first key of the given array without affecting the internal array pointer.
+	 *
+	 * @ticket 45055
+	 * @ignore
+	 * @since 5.9.0
+	 *
+	 * @param array $array An array.
+	 * @return string|int|null The first key of array if the array is not empty. NULL otherwise.
+	 */
+	function array_key_first( array $array ) {
+		foreach ( $array as $key => $value ) {
+				return $key;
+		}
+	}
+}
```

Comment:
- Suggest changing `$array` to `$arr`. Avoid using PHP keywords in a method/function signature due to PHP 8.1 and named key parameters in PHP 8.0. As a general best practice, try to avoid them in tests too.